torchtune/modules/tanh_gate.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
This feels like too much of a one-off to have in its own file

Reply:
I could call it activation_layers? I just thought to add it as a one off and group it when there's more modules that go with it.